test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -313,15 +322,15 @@ func testWorkspacesVirtualWorkspaces(t *testing.T, standalone bool) {
 				require.NoError(t, err, "failed to create root workspace roles")
 
 				err = server.virtualWorkspaceExpectations[0](func(w *tenancyv1beta1.WorkspaceList) error {
-					expectedOrgs := sets.NewString(orgName)
+					expectedOrgs := sets.NewString(orgName, "default")
```

Comment:
@davidfestal I needed this change to get the tests passing, but I'm assuming this is not desirable. Any idea what I might have done and how to fix it?

Reply:
welcome to shared test data (users with groups) 🎉

Workaround: one group membership per test that depends on it.

Reply:
actually even per-test users.

Reply:
One test did require a unique group to ensure auth isolation. I updated a different test that was using "root:default" to use a test org instead. PTAL